uvicorn/loops/asyncio.py

```diff
@@ -1,19 +1,7 @@
 import asyncio
-import platform
-import selectors
 import sys
 
 
 def asyncio_setup() -> None:  # pragma: no cover
-    loop: asyncio.AbstractEventLoop
-    if (
-        sys.version_info.major >= 3
-        and sys.version_info.minor >= 8
-        and platform.system() == "Windows"
-    ):
-        selector = selectors.SelectSelector()
-        loop = asyncio.SelectorEventLoop(selector)
-        asyncio.set_event_loop(loop)
-    else:
-        loop = asyncio.new_event_loop()
-        asyncio.set_event_loop(loop)
+    if sys.version_info >= (3, 8) and sys.platform == "win32":
+        asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

Comment:
is it the same as the deleted part ?

Reply:
it behaves the same, it's more like https://github.com/graingert/uvicorn/blob/avoid-deprecated-get-event-loop/uvicorn/loops/uvloop.py#L7 and works on py3.10

Reply:
ok, in essence the question was: we still force windows not to use the protractor event loop which is their default ?

Reply:
Yes

Reply:
ok so we're still NOT using the windows default with the chage, ie we are not on the protractor loop?

Reply:
oups I'm making double comment  sorry @graingert